cms-lite/src/main/resources/webapp/partials/widgets/string-resource-modal.html

```diff
@@ -9,7 +9,7 @@ <h4 class="modal-title" id="stringResourceLabel">{{msg('cmslite.header.resource'
             </div>
             <div class="modal-body">
                 <form id="stringResourceForm" name="stringResourceForm" class="form-horizontal"
-                      action="../cmsliteapi/resource/string/{{select.language}}/{{select.name}}" method="POST">
+                      action="{{ '../cmsliteapi/resource/string/' + select.language + '/' + select.name + '/' }}" method="POST">
```

Comment:
There was no slash at the end of this endpoint in the previous version. Why do we need one now?